schema/ContainerImage.schema.yaml

```diff
@@ -0,0 +1,102 @@
+title: ContainerImage
+'@id': stencila:ContainerImage
+extends: Thing
```

Comment:
I think this should extend https://schema.org/SoftwareApplication, or at least https://schema.org/CreativeWork. Those types already have many of the properties being added here e.g. `references` (aka `citations`), `downloadUrl`, `dateCreated`, `dateModified` etc

Reply:
I almost did this, but then looked at other similar examples and saw they extended thing. I'll make this change right away!